docs/it/week03/03.md

```diff
@@ -0,0 +1,27 @@
+---
+lang-ref: ch.03
+titolo: Week 3
+data traduzione: 30 marzo 2020
+traduzione: marcozullich
```

Comment:
Same for these two.
These are like `for`, `while`, `if` in a C program.
You cannot change them, haha.